File: agriTech/src/main/java/com/example/agriTech/dto/auth/LoginResponseDTO.java
package com.example.agriTech.dto.auth;

import lombok.Data;

@Data
public class LoginResponseDTO {
    private Long accountId;
    private String username;
    private String fullName;
    private Integer role;
    private String message;

    
    private String phone;    // Thêm cái này
    private String address;  // Thêm cái này
    
}

File: agriTech(DA3K)/app/src/main/java/com/example/agritechda3k/api/dto/LoginResponseDTO.kt
package com.example.agritechda3k.api.dto

data class LoginResponseDTO(
    val accountId:Long,
    val username:String,
    val fullName:String,
    val role:Int?,
    val message:String,

    val phone: String? = null,
    val address: String? = null,
)
